docs/pl/docs/tutorial/index.md

```diff
@@ -0,0 +1,81 @@
+# Samouczek - Instrukcja - Wprowadzenie
```

Comment:
I would change to simpler `# Samouczek - Wprowadzenie`, see my comment in `mkdocs.yml`